moped-editor/src/components/GridTable/Filters.js

```diff
@@ -123,6 +123,20 @@ const generateEmptyField = (uuid) => {
   return { ...defaultNewFieldState, id: uuid };
 };
 
+/**
+ * Returns whether the user input value for the filterParameter is valid
+ * @param {object} filterParameter
+ * @returns {boolean}
+ */
+const checkIsValidInput = (filterParameter) => {
```

Comment:
I see now that the existing validation was tied to this `filterParameter` object. I'll be taking a close look at the way this component uses its filter state this sprint so this is a nice preview! 🙏